.ansible-lint

```diff
@@ -5,3 +5,6 @@
 skip_list:
   - '106' # Role name {} does not match ``^[a-z][a-z0-9_]+$`` pattern
   - 'fqcn-builtins'
+
+warn_list:  # or 'skip_list' to silence them completely
```

Comment:
can you explain why you added this? 

I am rebasing your changes, so I will add this to the commit message.

Reply:
IMO a fail on an ``experimental`` lint feature is not feasible.

Its more a safety measure.